concepts/conditionals/about.md

```diff
@@ -1,3 +1,134 @@
 # About
 
-TODO: add information on conditionals concept
+## General Syntax
+
+A common way to conditionally execute logic in JavaScript is the if-statement. It consists of the `if` keyword, a condition wrapped in round brackets and a code block wrapped in curly brackets. The code block will only be executed if the condition evaluates to `true`.
+
+```javascript
+if (condition) {
+  // code that is executed if "condition" is true
+}
+```
+
+It can be used stand-alone or combined with the `else` keyword.
+
+```javascript
+if (condition) {
+  // code that is executed if "condition" is true
+} else {
+  // code that is executed otherwise
+}
+```
+
+## Nested If-Statements
+
+To nest another condition into the `else` statement you can use `else if`. Note that there is no `elseif` keyword in JavaScript. It is literally just `else` followed by another `if` statement.
```

Comment:
```suggestion
To nest another condition into the `else` statement you can use `else if`.
Note that there is no `elseif` keyword in JavaScript.
Instead write `else` followed by another `if` statement.
```

I try to avoid `just`, in this context.